site/source/tutorials/1010_containers_and_injecting_state/index.md

```diff
@@ -58,31 +56,35 @@ Currently, the application's `main` module is only responsible for creating the
 
 {% include_codefile 'demo/initial/biz-e-corp/src/main.ts' %}
 
-Now, we need to update `main` to construct and populate the `ApplicationContext` as well as register that context so that it is accessible to the rest of the application.
+Now, we need to update `main` to construct and populate the `ApplicationContext` create a `registry` for the application and use this to define the `ApplicationContext`. Finally in order to make the `registry` available within widget tree we need to pass the `registry` as a property to the `projector`.
```

Comment:
Now, we need to update `main` to construct and populate the `ApplicationContext` create a `registry` for the application and use this to define the `ApplicationContext`.   ... Missing a word and not particularly clear